geonode/settings.py

```diff
@@ -619,6 +619,15 @@
     os.getenv('DEFAULT_ANONYMOUS_DOWNLOAD_PERMISSION', 'True')
 )
 
+# Whether the remote resources should be public and downloadable by default
+# or not
+DEFAULT_ANONYMOUS_VIEW_PERMISSION_REMOTE = strtobool(
```

Comment:
We need this documented and a test case for it.

Also, would this be declared into the context processors too?